### DISCUSSION TITLE
Display PTY in different languages

### DISCUSSION THREAD

**Discussion by mtv-alex** [PE5PVB/TEF6686_ESP32#407] [Ideas]
I make a proposal to translate PTY codes in the language, in accordance with the established firmware language.
The standard IEC62106 has a special section with the translation of PTY codes into 20 languages.  I will attach the list of languages in the photo. For writing, we have to use the last column(16 character display).
If there is no translation table of PTY codes for the language in the receiver, then I suggest: leave the option in English or an independent translation by someone.
The tables can be found here: [PTY LANG.pdf](https://github.com/PE5PVB/TEF6686_ESP32/files/13339583/PTY.LANG.pdf)

![123](https://github.com/PE5PVB/TEF6686_ESP32/assets/147340942/7a9ea7c1-8706-42df-84da-9ddb69780a42)

**Comment by PE5PVB** [comment]
Yes I know. The main problem is that there is not enough RAM to store it for each language.

**Comment by lxsxl** [comment]
Thanks a lot for implementing PTY translations! :-)
